libs/platform/src/lib/value-help-dialog/components/define-tab/define-tab.component.html

```diff
@@ -23,7 +23,7 @@
                 colGrow
                 [class.fd-select-custom-class]="false"
                 #sttr
-                [compact]="true"
+                [compact]="isCompact"
```

Comment:
In all other components `contentDensity !== 'cozy'` used in template, and here getter is provided, let's unify the approach :)